### DISCUSSION TITLE
weird collisions with name

### DISCUSSION THREAD

**Discussion by rbarbrow** [microsoft/winget-cli#1623] [Q&A] [answered]
So I did a 

winget install microsoft.windowsterminal

and it returns 

"Multiple packages found matching input criteria. Please refine the input.
Name                    Id                        Source
---------------------------------------------------------
Hotspot Shield Free VPN 9WZDNCRDFNG7              msstore
Windows Terminal        Microsoft.WindowsTerminal winget"

how does whatever pos software like hotspot shield free VPN confuse winget

how is "microsoft.windowsterminal" not specific enough?

**Comment by Masamune3210** [comment, marked as answer]
Known issue with msstore backend, being worked on behind the scenes. Disable the msstore source with winget source remove msstore if you want to be rid of the error for now or specify the id with --id when you want to install something if you don't want to remove the source

**Reply by rbarbrow** [reply]
do you know the issue number so I can track it?

as long as its being worked on I will maintain the faith

I was just hoping it wasn't some weird msstore "SEO" where they were slipping in names to try to get more downloads or some other scummy bs

**Reply by denelon** [reply]
This [query](https://github.com/microsoft/winget-cli/issues?q=is%3Aissue+is%3Aopen+label%3Amsstore) lists all issues with the "msstore" label.
